contracts/payloads/IGP36/PayloadIGP36.sol

```diff
@@ -0,0 +1,752 @@
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Fluid
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Set weETHs token config and market rate curve on liquidity.
+        action1();
+
+        // Action 2: Deploy weETHs/wstETH.
+        action2();
+    }
+
+    function verifyProposal() external view {}
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Set weETHs token config and market rate curve on liquidity.
+    function action1() internal {
+        {
+            AdminModuleStructs.RateDataV1Params[]
+                memory params_ = new AdminModuleStructs.RateDataV1Params[](1);
+
+            params_[0] = AdminModuleStructs.RateDataV1Params({
+                token: weETHs_ADDRESS, // weETHs
+                kink: 80 * 1e2, // 80%
+                rateAtUtilizationZero: 0, // 0%
+                rateAtUtilizationKink: 5 * 1e2, // 5%
+                rateAtUtilizationMax: 100 * 1e2 // 100%
+            });
+
+            LIQUIDITY.updateRateDataV1s(params_);
+        }
+
+        {
+            AdminModuleStructs.TokenConfig[]
+                memory params_ = new AdminModuleStructs.TokenConfig[](1);
+
+            params_[0] = AdminModuleStructs.TokenConfig({
+                token: weETHs_ADDRESS, // weETHs
+                threshold: 0.3 * 1e2, // 0.3
+                fee: 10 * 1e2, // 10%
+                maxUtilization: 100 * 1e2
+            });
+
+            LIQUIDITY.updateTokenConfigs(params_);
+        }
+    }
+
+    /// @notice Action 2: Deploy weETHs/wstETH.
+    function action2() internal {
+        VaultConfig memory vaultConfig = VaultConfig({
+            // user supply config for the vault on Liquidity Layer.
+            supplyToken: weETHs_ADDRESS,
+            supplyMode: 1, // Mode 1
+            supplyExpandPercent: 25 * 1e2, // 25%
+            supplyExpandDuration: 12 hours, // 12 hours
+            supplyBaseLimitInUSD: 0, //
+            supplyBaseLimit: 4000 * 1e18, // 4000 weETHs
+            borrowToken: wstETH_ADDRESS,
+            borrowMode: 1, // Mode 1
+            borrowExpandPercent: 20 * 1e2, // 20%
+            borrowExpandDuration: 12 hours, // 12 hours
+            borrowBaseLimitInUSD: 7_500_000, // $7.5M
+            borrowBaseLimit: 0,
+            borrowMaxLimitInUSD: 200_000_000, // $200M
```

Comment:
> Max borrowing limit: 4000 wstETH

this should be here ~10M USD

base borrow limit maybe start lower here at 5M USD then? 

supplyBaseLimit also ~5M USD?